azavea/raster-foundry
# Remove placeholders + fix scene removal

## Overview

This PR makes two UI changes.

1) Removes all placeholder images and replaces with a placeholder `div` that can have our custom text within. It seems better to control the content of a placeholder or have none at all rather than showing the dimensions of the placeholder (which were oftentimes distorted). This also removes any extraneous requests and dependency on an outside service.

2) Makes the 'Remove from project' link on the scene detail page within the library/project context actually work.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

<img width="1280" alt="screen shot 2017-02-24 at 10 38 11 am" src="https://cloud.githubusercontent.com/assets/2442245/23310446/8dc6c37a-fa80-11e6-89b3-58711fc11a16.png">
<img width="1262" alt="screen shot 2017-02-24 at 10 38 19 am" src="https://cloud.githubusercontent.com/assets/2442245/23310447/8dcc6bcc-fa80-11e6-9bac-d1698b70cffd.png">
<img width="1279" alt="screen shot 2017-02-24 at 10 55 15 am" src="https://cloud.githubusercontent.com/assets/2442245/23310448/8dd75cb2-fa80-11e6-8cd8-9a156f52a482.png">

### Notes

In cases where we depend on image dimensions to take up layout space rather than having dimensions determined via css, you can do an inline style to set the desired height/width of the placeholder or create an additional class to enforce the dimensions. I know inline styles are "bad" but being that our placeholder urls contained the dimensions in these cases I think it's a reasonable solution.

## Testing Instructions

 1) Testing placeholders
    - Look through the UI where items have thumbnails (scene browse, tool catalog), ensuring that layout isn't affected by placeholders.
    - Look at network requests and ensure no requests to 'placehold.it' are being made

  2) Testing scene removal
    - Go to your library -> projects and select a project with scenes in it
    - Click on a scene to go to the details page, ensure the 'Remove from project' button works (and no error is thrown in the console).

Connects #1100
Connects #1124
